feature/contributors/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/contributors/ContributorsViewModel.kt

```diff
@@ -13,10 +16,15 @@ import kotlinx.coroutines.flow.stateIn
 
 @HiltViewModel
 class ContributorsViewModel @Inject constructor(
-    contributorsRepository: ContributorsRepository
-) : KmpViewModel() {
+    contributorsRepository: ContributorsRepository,
+    val userMessageStateHolder: UserMessageStateHolder,
+) : KmpViewModel(), UserMessageStateHolder by userMessageStateHolder {
     private val contributors = contributorsRepository
         .contributors()
+        .handleErrorAndRetry(
+            AppStrings.Retry,
```

Comment:
[nit] Sorry for sudden comments 🙇 As a beginner of codebase, It's nice to have a arguments name for the method calling. & this comments should not be blocker for the PR merging. 
Just a randomly comments 🙇

Reply:
You have a point. But maybe the rest of the code for handleErrorAndRetry doesn't use named arguments. Maybe we can work on another issue.